node/Makefile

```diff
@@ -0,0 +1,7 @@
+.PHONY: install test_ci
+
+install:
+	npm install
+
+test_ci:
```

Comment:
can we also add `test:` that runs `npm test`.

If we start a `cd folder && make test` pattern for local vs `cd folder && make test_ci` pattern for CI that might be nice.